### PR TITLE
CCM-12624-Added ccm-smp docs

### PR DESCRIPTION
**Pull request by charanya-j**
AWS SMP doc and the CCM SMP roadmap

### CODE REVIEW

File: docs/cloud-cost-management/2-getting-started-ccm/ccm-smp/aws-smp.md [hunk 1]
```diff
@@ -0,0 +1,885 @@
...
+- smtp-secret - Required to support budget alerts email.
+
+```
+SMTP_HOST: <SMTP_HOST>
+SMTP_PASSWORD: <SMTP_PASSWORD>
+SMTP_USERNAME: <SMTP_USERNAME>
+```
+
+:::important
+`kubectl edit pvc wal-volume-harness-timescaledb-0 -n <namespace>` and increase to 100Gi. It is used by recommendations and anomalies features within CCM services.
```
Comment (charanya-j) [inline docs/cloud-cost-management/2-getting-started-ccm/ccm-smp/aws-smp.md:884]: I couldn't understand this note. Could you please explain?

File: docs/cloud-cost-management/2-getting-started-ccm/ccm-smp/1-smp-ccm-roadmap.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: Roadmap and feature availability
+description: The features available and the roadmap for the self-Managed Enterprise edition of Harness Cloud Cost Management.
+# sidebar_position: 10
+---
+
+
+# Self-Managed Enterprise Edition - Roadmap and feature availability
+The Harness Self-Managed Enterprise Edition of Cloud Cost Management is a comprehensive solution designed to optimize your cloud resources and minimize expenses. It provides granular insights into your cloud expenses, including detailed breakdowns and cost allocation. By leveraging the cost reporting features like Perspectives and dashboards, CCM empowers you to make informed business decisions that align with your strategic objectives.
```
Comment (SudheendraKatte) [inline docs/cloud-cost-management/2-getting-started-ccm/ccm-smp/1-smp-ccm-roadmap.md:9]: Please check capitalization of "Perspectives and dashboards". If it is UI element, making it bold would be a good idea.

**Review by SudheendraKatte — DISMISSED** [review]
Approved with a minor suggestion.

File: docs/cloud-cost-management/2-getting-started-ccm/ccm-smp/1-smp-ccm-roadmap.md (hunk 2 above)
Comment (charanya-j) [reply docs/cloud-cost-management/2-getting-started-ccm/ccm-smp/1-smp-ccm-roadmap.md:9]: These are CCM features. Though they are present on the UI, in this paragraph, I'm not referring to a specific page or screen. I should change P in Perspectives to lowercase. Thank you.

**Pull request merged**